recipe/meta.yaml

```diff
@@ -28,8 +28,12 @@ requirements:
     - numpy >=1.12.1
     - protobuf >=3.4.0
     - six >=1.10.0
-    - tensorboard >=1.5.0,<1.6.0
     - backports.weakref >=1.0rc1
+    - astor
+    - grpcio
+    - termcolor
+    - gast
```

Comment:
These new requirements should all have lower version bounds as specified in the projects setup.py specifically:
```
astor >=0.6.0
grpcio >=1.8.6
termcolor >=1.1.0
gast >=0.2.0